### PR TITLE
vendors stack: support multiple domains & maintenance mode

### PR DESCRIPTION
**Pull request by YuryHrytsuk**
## What do these changes do?
* add multiple-domains support for vendors manual
* include vendors manual in maintenance
* add j2cli customization support 

## Related issue/s
* closes https://github.com/ITISFoundation/osparc-ops-environments/issues/885

## Related PR/s
* https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/merge_requests/1100

## Checklist
- [ ] I tested and it works

### CODE REVIEW

File: services/vendors/j2cli_customization.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+def _generate_vendors_manual_traefik_rule(domains: str, subdomain_prefix: str) -> str:
+    domain_list = domains.strip().strip(",").split(",")
```
Comment (mrnicegyu11) [inline services/vendors/j2cli_customization.py:2]: Can we add assertions here, or even pydantic checks? Would be nice

File: services/vendors/docker-compose.yml.j2 [hunk 2]
```diff
@@ -28,7 +30,7 @@ services:
         - traefik.http.services.vendor_manual.loadbalancer.server.port=${VENDOR_MANUAL_PORT}
         - traefik.http.routers.vendor_manual.entrypoints=https
         - traefik.http.routers.vendor_manual.tls=true
-        - traefik.http.routers.vendor_manual.rule=Host(`${VENDOR_MANUAL_DOMAIN}`)
+        - traefik.http.routers.vendor_manual.rule={{ generate_vendors_manual_traefik_rule(VENDOR_MANUAL_PRODUCTS, VENDOR_MANUAL_SUBDOMAIN_PREFIX) }}
```
Comment (mrnicegyu11) [inline services/vendors/docker-compose.yml.j2:33]: Nice!

**Review by mrnicegyu11 — APPROVED** [review]
Nice find with the customization, fits good with our architecture. We could use this in more places actually. Didnt know this existed, thanks a lot

:--)

File: services/vendors/j2cli_customization.py (hunk 1 above)
Comment (YuryHrytsuk) [reply services/vendors/j2cli_customization.py:2]: What do you mean? I am not sure I get the idea

Comment (YuryHrytsuk) [reply services/vendors/j2cli_customization.py:2]: as we discussed, it is a good idea to implement config validation. We can look into it later :+1:

**Pull request merged**